src/modules/quick_method_module.f90

```diff
@@ -755,6 +756,13 @@ subroutine read_quick_method(self,keywd,ierr)
             if (index(keyWD,'COARSEINT').ne.0) self%coarse_cutoff=.true.
             if (index(keyWD,'TIGHTINT').ne.0) self%tight_cutoff=.true.
 
+            ! read and enable exporting
+            if (index(KeyWD,'EXPORT=') .ne. 0) then
+                tempstring=' '
+                call read(keyWD, 'EXPORT', tempstring)
+                if(index(tempstring,'MOLDEN') /= 0) write_molden=.true.
```

Comment:
Should we throw an error if there is a typo/unknown export format?

Reply:
Fixed.